app/models/timeline.rb

```diff
@@ -9,11 +9,12 @@ class Timeline
     initialContactDate: "Initial contact from complaint"
   }.with_indifferent_access.freeze
 
-  def initialize(complaint_attributes, tta_reports)
+  def initialize(complaint_attributes, tta_reports = [], monitoring_reviews = [])
```

Comment:
[suggestion] i don't think we need the default values; they are both being fetched with a `.where`, which returns an empty array if nothing is found.

Reply:
That makes sense. I added them mostly because of the test suites trying to start up a timeline with only tta, but that's kind of an edge case so I might as well drop this